src/widgets/chart.rs

```diff
@@ -745,10 +745,12 @@ impl<'a> Chart<'a> {
             if let Some(inner_width) = legends.clone().max() {
                 let legend_width = inner_width + 2;
                 let legend_height = legends.count() as u16 + 2;
+                #[allow(deprecated)]
```

Comment:
Add an issue to fix this up

Reply:
Thanks for opening an issue: https://github.com/ratatui-org/ratatui/issues/793